pdfannots/printer/todocsv.py

```diff
@@ -0,0 +1,25 @@
+from .csv import CsvPrinter
+
+import typing
+from ..types import Annotation, Document
+
+
+class TodocsvPrinter(CsvPrinter):
+    """
+    Create a todo-list of all comments to keep track, which comments were addressed
```

Comment:
This one seems to me too specialised to make sense maintaining in the base project. I wonder if you can put it in a small external script that imports pdfannots? I'm happy to add extensibility hooks if necessary for something like that.